pkg/persistentvolumeclaim/debug_cstor.go

```diff
@@ -0,0 +1,198 @@
+/*
+Copyright 2020-2021 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package persistentvolumeclaim
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/fatih/color"
+
+	cstortypes "github.com/openebs/api/v2/pkg/apis/types"
+	"github.com/openebs/openebsctl/pkg/client"
+	"github.com/openebs/openebsctl/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/printers"
+)
+
+// DebugCstorVolumeClaim is used to debug a cstor volume by calling various modules
+func DebugCstorVolumeClaim(k *client.K8sClient, pvc *corev1.PersistentVolumeClaim, pv *corev1.PersistentVolume) error {
+	var cstorResources util.CstorVolumeResources
+	cstorResources.PVC, _ = k.GetPVC(pvc.Name, "default")
+	cstorResources.PV = pv
+	if pv != nil {
+		cv, _ := k.GetCV(pv.Name)
```

Comment:
That would be handled in the specific modules. The reason for swallowing error is that we would show the status of the whatever is present rest of the things would be shown as `Not found`.

Reply:
Also, only one one possible error can be got here i.e its not found, and in that case the CR struct will be nil. This would save some cyclomatic complexity.